### PR TITLE
Fix for JBEAP-5175

### PR DESCRIPTION
**Pull request by jbertram**

### CODE REVIEW

File: artemis-server/src/main/java/org/apache/activemq/artemis/core/config/ClusterConnectionConfiguration.java [hunk 1]
```diff
@@ -178,6 +262,15 @@ public ClusterConnectionConfiguration setConfirmationWindowSize(int confirmation
       return this;
    }
 
+   public int getProducerWindowSize() {
+      return producerWindowSize;
+   }
+
+   public ClusterConnectionConfiguration setProducerindowSize(int producerWindowSize) {
```
Comment (jmesnil) [inline artemis-server/src/main/java/org/apache/activemq/artemis/core/config/ClusterConnectionConfiguration.java:269]: There is a missing `W` in the method name.

**Comment by clebertsuconic** [comment]
@jbertram  just fixed the missing W upstream, we need to update the branch now.

**Comment by andytaylor** [comment]
cheers,

Jeff how does this effect the subsystem, can it wait for 7.0.3 or do you
need a new release? If the latter is anyone available to do this on Monday?

**Comment by andytaylor** [comment]
or I can do it tomorrow?

**Comment by jmesnil** [comment]
There is no hurry. That can wait the next release

## 

Jeff Mesnil
jmesnil@gmail.com
http://jmesnil.net/

**Pull request merged**